sequencer.c

```diff
@@ -2346,13 +2347,21 @@ static int read_and_refresh_cache(struct repository *r,
 			_(action_name(opts)));
 	}
 	refresh_index(r->index, REFRESH_QUIET|REFRESH_UNMERGED, NULL, NULL, NULL);
+
 	if (index_fd >= 0) {
 		if (write_locked_index(r->index, &index_lock,
 				       COMMIT_LOCK | SKIP_IF_UNCHANGED)) {
 			return error(_("git %s: failed to refresh the index"),
 				_(action_name(opts)));
 		}
 	}
+
+	/*
+	 * If we are resolving merges in any way other than "ort", then
+	 * expand the sparse index.
+	 */
+	if (!opts->strategy || strcmp(opts->strategy, "ort"))
```

Comment:
Did you mean `if (opts->strategy && strcmp(opts->strategy, "ort")`? That would be the inversion of what is in line 639...

Reply:
Good point.

Reply:
> 
> 
> Good point. It was unclear to me about what to do when `opts->strategy` was `NULL`, but this makes sense to use that existing example.

It depends on the default for `opts->strategy` when it is set to `NULL`. In `microsoft/git`, we decided that it should be `ort` ;-)